src/components/top-bar-equipment-seach-dialog/top-bar-equipment-search-dialog.tsx

```diff
@@ -54,19 +60,40 @@ export const TopBarEquipmentSearchDialog: FunctionComponent<
     const enableSearchDialog = useCallback(() => {
         setIsDialogSearchOpen(true);
     }, [setIsDialogSearchOpen]);
+    const { snackWarning } = useSnackMessage();
 
     const onSelectionChange = useCallback(
         (element: EquipmentInfos) => {
             setIsDialogSearchOpen(false);
             updateSearchTerm('');
             addToLocalStorageSearchEquipmentHistory(studyUuid, element);
-            showVoltageLevelDiagram(element);
+            fetchNetworkElementInfos(
+                studyUuid,
+                currentNode?.id,
+                element.type,
+                EQUIPMENT_INFOS_TYPES.MAP.type,
```

Comment:
I'm not sure "MAP" is the right type
when checking "network-map-server", this type is not often implemented and will throw errors most of the time
"LIST" might be better ?

Reply:
done